README.md

```diff
@@ -1,33 +1,26 @@
 # BWLTool
 Tool zur Hilfe bei häufigen Rechnungen in der BWL.
 
-<H2>Was macht unser Tool?</H2>
- - Es wird Port 8080 für einen Tomcat Webserver geöffnet</br>
- - Es wird mit Vaadin + Spring Boot eine Webseite verfügbar gemacht, mit der man häufige BWL Rechnungen effizent lösen kann
+## Was macht unser Tool?
+- Es wird Port 8080 für einen Tomcat Webserver geöffnet
+- Es wird mit Vaadin + Spring Boot eine Webseite verfügbar gemacht, mit der man häufige BWL Rechnungen effizent lösen kann
 
-<H2>Richtig in der IDE zum laufen bringen</H2>
+## Richtig in der IDE zum laufen bringen
+1. Die pom.xml mit IntelliJ oder Eclipse öffnen
+2. Abwarten bis alle Depencencies heruntergeladen wurden
+3. Am Besten mit Java 11 starten
```

Comment:
Sure 11 is better? It was build for the vaadin website for 15

Reply:
From pom.xml:
```xml
<properties>
    <java.version>11</java.version>
```
Maven builds this project with Java 11.

I don't care which Java version is used, but it should be consistent.

Reply:
Okay, akward that it is generated like this...